bin/coq/coqtop.ml

```diff
@@ -93,17 +93,24 @@ let term =
                   ~dir_contents:dc theory coq_module
               , theory.buildable.use_stdlib
               , theory.buildable.coq_lang_version
-              , Dune_rules.Coq_lib_name.wrapper (snd theory.name) )
+              , Dune_rules.Coq_lib_name.wrapper (snd theory.name)
+              , theory.buildable.mode )
             | Some (`Extraction extr) ->
               ( Dune_rules.Coq_rules.coqtop_args_extraction ~sctx ~dir extr
                   coq_module
               , extr.buildable.use_stdlib
               , extr.buildable.coq_lang_version
-              , "DuneExtraction" )
+              , "DuneExtraction"
+              , extr.buildable.mode )
```

Comment:
No idea, I never actually used extraction. But I don't see why it shouldn't, and it felt like it would make sense to use the information since we have it.

Reply:
Should we do anything here @Alizter?

Reply:
No nothing actionable, just a question.